packages/flutter/lib/src/material/elevated_button.dart

```diff
@@ -393,7 +395,7 @@ class _ElevatedButtonWithIcon extends ElevatedButton {
          focusNode: focusNode,
          autofocus: autofocus ?? false,
          clipBehavior: clipBehavior ?? Clip.none,
-         child: _ElevatedButtonWithIconChild(icon: icon, label: label),
+         child: _ElevatedButtonWithIconChild(icon: icon, label: label, iconAlignment: iconAlignment ?? ButtonIconAlignment.start),
```

Comment:
can the default just be in the constructor?

Reply:
i have it as required so to be able to accomplish what is requested I have to make the value optional, do I proceed?

Reply:
If it's required then you don't need to use `??`, right? What is the API you intend to expose?

Reply:
I said it wrong, I mean required in `_ElevatedButtonWithIconChild` but not in `_ElevatedButtonWithIcon` or `ElevatedButton.icon` which makes it mandatory somewhere to check for `null` using `??`.

Reply:
What do you suggest doing?